Arrays/reshapemat.py

```diff
@@ -0,0 +1,36 @@
+def reshape(mat,a,b):
+    '''for i in range(a):
+        j=i
+        for j in range(b):
+            print(mat[j])'''
+    flat=[]
+    for ele in mat:
+        flat.extend(ele)
+     
+    for i in range(a*b):
+        if i%b==0and i!=0:
+            print("")
+        print(flat[i],end=" ")
+        
+        
+        
+m,n=input().split()
+a,b= input().split()
+mat=[]
+m=int(m)
```

Comment:
In lines 20-23, you can do unpacking. Something like,
m, n, a, b = int(m), int(n), int(a), int(b)